tests/agent/jobs/test_job_processing.py

```diff
@@ -0,0 +1,96 @@
+import os
+import time
+
+import pytest
+import requests
+
+org_id = os.getenv("GX_CLOUD_ORGANIZATION_ID")
+token = os.getenv("GX_CLOUD_ACCESS_TOKEN")
+gql_url = url = f"http://localhost:5000/organizations/{org_id}/graphql"
+
+
+@pytest.fixture(scope="session")
+def wait_for_docker_compose():
+    # pretest setup
+    get_agent_status_body = """
+        query agent_status {
+            agentStatus {
+            active
+            __typename
+            }
+        }
+    """
+    response = requests.post(
+        url=url,
+        json={"query": get_agent_status_body},
+        headers={
+            "Content-Type": "application/json",
+            "Authorization": f"Bearer {token}",
+        },
+    )
+    response.raise_for_status()
+
+    if not response.json()["data"]["agentStatus"]["active"]:
+        return
+    ## check here to see if the agent is online
+
+    yield
+    print("Post pytest test_job_processing teardown")
+    # posttest teardown
+
+
+def test_job_processing(wait_for_docker_compose):
+    body = """
+      mutation createRunCheckpointJob($checkpointId: UUID!) {
+        createRunCheckpointJob(checkpointId: $checkpointId) {
+          jobId
+          __typename
+        }
+      }
+      """
+
+    variables = """
+      {"checkpointId": "5f3814d6-a2e2-40f9-ba75-87ddf485c3a8"}
+      """
+
+    response = requests.post(
+        url=gql_url,
+        json={"query": body, "variables": variables},
+        headers={"Content-Type": "application/json", "Authorization": f"Bearer {token}"},
```

Comment:
```suggestion
        headers={"Authorization": f"Bearer {token}"},
```
You don't need the content type if you are using the `json` keyword `request.post(..., json=...)`.
`requests` adds the content-type automatically.